coprocessor-cache.md

```diff
@@ -36,4 +36,26 @@ You can configure Coprocessor Cache via the `tikv-client.copr-cache` configurati
 
 ## Check the cache effect
 
-Currently (TiDB v4.0.0-rc.2), Coprocessor cache is still an experimental feature. Users cannot learn how many push-down requests hit the cache in a SQL statement, or understand the overall cache hits. The corresponding monitoring and checking method will be introduced in later TiDB versions.
+You can check the cache effect of Coprocessor by executing `EXPLAIN ANALYZE` or viewing the Grafana monitoring panel.
+
+### Use `EXPLAIN ANALYZE`
+
+Users can view the cache hit rate in [Operators for accessing tables](/choose-index.md#operators-for-accessing-tables) through [statement `EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md). See the following example:
```

Comment:
To keep the person style consistent in this document. No need to strictly follow the Chinese version.
```suggestion
You can view the cache hit rate in [Operators for accessing tables](/choose-index.md#operators-for-accessing-tables) by using the [`EXPLAIN ANALYZE` statement](/sql-statements/sql-statement-explain-analyze.md). See the following example:
```

Reply:
OK, thanks:)